clients/src/main/resources/common/message/ListTransactionsRequest.json

```diff
@@ -0,0 +1,31 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "apiKey": 66,
+  "type": "request",
+  "listeners": ["zkBroker", "broker"],
+  "name": "ListTransactionsRequest",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  "fields": [
+    { "name": "StatesFilter", "type": "[]string", "versions": "0+",
+      "about": "The transaction states to filter by: if empty, all transactions are returned; if non-empty, then only transactions matching one of the filtered states will be returned"
+    },
+    { "name": "ProducerIdFilter", "type": "[]int64", "versions": "0+",
+      "about": "The producerIds to filter by: if empty, no transactions will be returned; if non-empty, only transactions which match one of the filtered producerIds will be returned"
```

Comment:
`no transactions will be returned` 
^^^^^^^^^^^^^^^^^^^^^^^^^^ this document is a bit weird since the code shows that all transactions are returned if this filter is empty.

Reply:
Yeah, my bad. On a side note, do you think it would be clearer if we used `null` to indicate the absence of the filter?

Reply:
> On a side note, do you think it would be clearer if we used null to indicate the absence of the filter?

it should be fine in this case. However, the `empty` collection gets weird as it absolutely gets nothing :(